<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarGUIClass</class>
 <widget class="QWidget" name="StarGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarGUI</string>
  </property>
  <widget class="QTableView" name="Table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CoordinatesRA_textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="CoordinatesRA_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CoordinatesRA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>See start which are in studing constelation</string>
   </property>
  </widget>
  <widget class="QLabel" name="CoordinatesDec_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CoordinatesDec</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CoordinatesDec_textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Search_textEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StarGUI.qrc"/>
 </resources>
 <connections/>
</ui>
